k8s/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
A Secret resource is probably the better choice for this example: https://kubernetes.io/docs/concepts/configuration/secret/.